recipes/deblur/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
   name: deblur
-  version: "0.1.8"
+  version: "1.0.0"
 
 source:
   git_url: https://github.com/biocore/deblur.git
-  git_rev: 0.1.8
+  git_rev: 1.0.0
 
 build:
-  number: 2
+  number: 3
```

Comment:
Although it is fine to start with an arbitrary number, for every version, the build number should start with 0. This is minor. I will leave to you whether you wanna change it.